src/nft/components/profile/view/ViewMyNftsAsset.tsx

```diff
@@ -1,14 +1,21 @@
 import { Trans } from '@lingui/macro'
 import { MouseoverTooltip } from 'components/Tooltip'
+import Tooltip from 'components/Tooltip'
 import { Box } from 'nft/components/Box'
 import * as Card from 'nft/components/collection/Card'
 import { AssetMediaType } from 'nft/components/collection/Card'
 import { bodySmall } from 'nft/css/common.css'
 import { themeVars } from 'nft/css/sprinkles.css'
 import { useBag, useIsMobile, useSellAsset } from 'nft/hooks'
 import { TokenType, WalletAsset } from 'nft/types'
+import { useEffect, useState } from 'react'
 import { useMemo } from 'react'
 import { useNavigate } from 'react-router-dom'
+
+const TOOLTIP_TIMEOUT = 2000
+const ADDED_TO_BAG_TOOLTIP_TEXT = 'Selected'
+const REMOVED_FROM_BAG_TOOLTIP_TEXT = 'Deselected'
```

Comment:
nit: rename this to SELECT and DESELECT_TOOLTIP_TEXT

Reply:
done